yt_dlp/extractor/common.py

```diff
@@ -3189,7 +3189,8 @@ def parse_content_type(content_type):
                 return f
             return {}
 
-        def _media_formats(src, cur_media_type, type_info={}):
+        def _media_formats(src, cur_media_type, type_info=None):
```

Comment:
Are we mutating type_info? I don't see where. Though, u can keep this change anyway

Reply:
No, but just to make sure (since it's a widely known bad practices in Python)